packages/client/src/components/Dashboard/TeamMembers.jsx

```diff
@@ -0,0 +1,204 @@
+      setShowAddModal(false);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  const handleEditSave = async () => {
+    try {
+      const response = await fetch(`${apiURL()}/teamMembers/${editMemberId}`, {
+        method: 'PATCH',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          first_name: editMemberFirstName,
+          last_name: editMemberLastName,
+        }),
+      });
+      if (!response.ok) {
+        throw new Error('Failed to edit team member');
+      }
+      fetchTeamMembers();
+      setEditMemberId(null);
+      setEditMemberFirstName('');
+      setEditMemberLastName('');
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  return (
+    <div className="team-members-container">
+      <div className="team-members-heading-container">
+        <Typography className="teamMembers-heading" variant="h6">
+          Team Members
+        </Typography>
+        <Button
+          type="button"
+          variant="contained"
+          size="small"
+          onClick={() => setShowAddModal(true)}
+        >
+          Add Team Member
+        </Button>
+      </div>
+      <div className="add-member">
+        <AddTeamMemberModal
+          showAddModal={showAddModal}
+          setShowAddModal={setShowAddModal}
+          newMemberFirstName={newMemberFirstName}
+          setNewMemberFirstName={setNewMemberFirstName}
+          newMemberLastName={newMemberLastName}
+          setNewMemberLastName={setNewMemberLastName}
+          handleAddMember={handleAddMember}
+        />
+      </div>
+
+      {editMemberId && (
+        <div className="edit-member-container">
+          <Typography variant="body1">Edit Member</Typography>
+          <div className="edit-member-details">
+            <div className="input-container">
+              <input
+                type="text"
+                value={editMemberFirstName}
+                onChange={(e) => setEditMemberFirstName(e.target.value)}
+                placeholder="First Name"
+              />
+              <input
+                type="text"
+                value={editMemberLastName}
+                onChange={(e) => setEditMemberLastName(e.target.value)}
+                placeholder="Last Name"
+              />
+            </div>
+            <div className="button-container">
+              <button
+                className="save-button"
+                type="button"
+                onClick={handleEditSave}
+              >
+                Save
+              </button>
+              <button
+                className="cancel-button"
+                type="button"
+                onClick={() => setEditMemberId(null)}
+              >
+                Cancel
+              </button>
+            </div>
+          </div>
+        </div>
+      )}
+
+      <div>
+        <ul>
+          {teamMembers.map((member) => (
+            <li key={member.member_id}>
```

Comment:
This would make an amazing React component!
```suggestion
           <TeamMemberListItem 
             key={member.member_id}
             id={member.member_id}
             firstName={...}
             lastName={...}
             ...
           />
```